ui/src/react-admin/modules/content-page/components/blocks/BlockSpotlight/BlockSpotlight.editorconfig.ts

```diff
@@ -58,7 +58,6 @@ export const SPOTLIGHT_BLOCK_CONFIG = (position = 0): ContentBlockConfig => ({
 			title: TEXT_FIELD(undefined, {
 				label: tText('admin/content-block/helpers/generators/spotlight___titel'),
 				editorType: ContentBlockEditor.TextInput,
-				validator: undefined,
```

Comment:
Removing this makes the field required, intentional?

Reply:
no, that wasn't my intention
i'll revert it

we should change that at some point. Validator not set should equal validator undefined and should mean no validation.